packages/validations/src/validations/userRoutesValidator.ts

```diff
@@ -1,81 +1,100 @@
 import { z } from 'zod';
 
-const JoiObjectId = z.string().regex(/^[0-9a-fA-F]{24}$/g);
+const emailValidator = z
+  .string()
+  .email()
+  .transform((str) => str.trim().toLowerCase());
+
+const passwordValidator = z
+  .string()
+  .min(7)
+  .refine((str) => !str.includes('password'), {
```

Comment:
Is there any reason why we shouldn't allow the password to contain the word "password"?

If not, can we instead check whether the password contains at least one uppercase letter, at least one number, at least one non-alphanumeric character and should not match the username?